geos-mesh/src/geos/mesh/processing/CreateConstantAttributePerRegion.py

```diff
@@ -13,7 +14,7 @@
     vtkDataSet,
 )
 
-from geos.utils.Logger import ( getLogger, Logger, logging, CountWarningHandler )
+from geos.utils.Logger import getLogger, Logger, CountWarningHandler
```

Comment:
Let's keep parenthesis when doing multiple import other than standard

```suggestion
from geos.utils.Logger import ( getLogger, Logger, CountWarningHandler )
```